ci-operator/config/red-hat-data-services/ods-ci/red-hat-data-services-ods-ci-master__rhoai2.13-ocp4.16-lp-interop.yaml

```diff
@@ -0,0 +1,66 @@
+base_images:
+  cli:
+    name: "4.16"
```

Comment:
iiuc, this OCP 4.16 job (as well as 4.18) was ran using an OCP 4.13 ods-ci image build?
 https://github.com/red-hat-data-services/ods-ci/blob/93a2c682f12e1526a07306af764997d9356afd54/ods_ci/build/Dockerfile_interop#L11

Reply:
please resolve if this is verified